packages/ai-constructs/src/conversation/runtime/types.ts

```diff
@@ -24,4 +30,23 @@ export type ConversationTurnEvent = {
     };
   };
   messages: Array<ConversationMessage>;
+  toolsConfiguration?: {
+    tools: Array<{
```

Comment:
nit: complex array type should be defined like above in line 32

Reply:
I refrained from that because we don't expect customer to create this and we'd rather not eagerly add public types that are useless to customer and pollute intellisense. I.e. we'd rather have `API.md` less verbose than absolutely necessary.

fwiw for internal consumption I did extract this type here https://github.com/aws-amplify/amplify-backend/blob/c04decbe0e8067bd2b800ecd1235c52aa886334e/packages/ai-constructs/src/conversation/runtime/event-tools-provider/types.ts#L3-L5 .

Reply:
I see